### ISSUE TITLE
Fix eslint issues in JS files.

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov_base#676]

**Comment by tonypaulbarker** [comment]
@markconroy I think it would be a handy practice to post linting errors to individual linting issues when they arise so the reviewer can see a record of the errors the developer saw before the fixes were applied. What do you think?

**Comment by tonypaulbarker** [comment]
I had trouble running ESLint. Opened https://github.com/localgovdrupal/localgov_base/issues/701

**Comment by finnlewis** [comment]
@tonypaulbarker we've only just started running the automated eslint checks on our github actions, so there are quite a lot of errors. See https://github.com/localgovdrupal/localgov_base/actions/runs/12793150319/job/36640979598?pr=699#step:4:123

```
✖ 103 problems (87 errors, 16 warnings)
  76 errors and 0 warnings potentially fixable with the `--fix` option.
```

So it is probably more efficient to address as many as we can first, then spin off individual issues where things get more involved on particular errors.

I'll create a branch off this issue to try the automated fixing and see how far we get.

**Comment by finnlewis** [comment]
My bad, this has already been done it seems. 


https://github.com/localgovdrupal/localgov_base/pull/689
https://github.com/localgovdrupal/localgov_base/pull/690
https://github.com/localgovdrupal/localgov_base/pull/692
https://github.com/localgovdrupal/localgov_base/pull/693

**Comment by markconroy** [comment]
Yep, this repo should be fairly clean for JS linter issues at this stage.

If anything new pops up, create an issue and file a PR if possible.
